go/oasis-node/cmd/common/metrics/mem.go

```diff
@@ -0,0 +1,98 @@
+// Package metrics implements a prometheus metrics service.
+package metrics
+
+import (
+	"fmt"
+	"os"
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/procfs"
+)
+
+const (
+	MetricMemVmSizeBytes   = "oasis_worker_mem_VmSize_bytes" // nolint: golint
```

Comment:
Why is `// nolint: golint` needed? Missing documentation? should just add it :)

Reply:
golint complains `MetricMemVmSizeBytes` should be named capital `VM`, but this contradicts the /proc/<pid>/stats naming.

Reply:
maybe also add a note for this in code